pubspec.yaml

```diff
@@ -24,7 +24,7 @@ dependencies:
   os_detect: ^2.0.1
 
   # From Flutter Community: Provides an API for querying information about an application package.
-  package_info_plus: ^4.0.1
+  package_info_plus: ^5.0.1
```

Comment:
@Zazo032 Do you realize that package_info_plus requires Dart 3.2.0+ and that making this change would require all users of upgrader to upgrader to Flutter 3.16.0? What is in package_info_plus 5.0.1 that you need right now?

Reply:
@Zazo032 Does `'>=4.0.1 <6.0.0'` work?

Reply:
Yes, `'>=4.0.1 <6.0.0'` would work, I'll change that 😄